src/platform/common/utils/localize.ts

```diff
@@ -77,25 +80,28 @@ export namespace GitHubIssue {
         'Please provide a descriptive title summarizing your issue.'
     );
     export const titlePlaceholder = localize(
-        'GitHubIssue.titlePlaceholder',
+        { key: 'GitHubIssue.titlePlaceholder', comment: ['{Locked="kernel"}'] },
```

Comment:
I am assuming we want to keep `kernel` as is and not change it from language to language.